ecosystem/typescript/sdk/src/tests/e2e/indexer.test.ts

```diff
@@ -0,0 +1,75 @@
+import { AptosAccount } from "../../aptos_account";
+import { AptosClient } from "../../providers/aptos_client";
+import { bcsSerializeBool } from "../../bcs";
+import { FaucetClient } from "../../providers/faucet_client";
+import { IndexerClient } from "../../providers/indexer";
+import { TokenClient } from "../../token_client";
+import { API_TOKEN, longTestTimeout } from "../../utils/test_helper.test";
+import { Network, NetworkToIndexerAPI, NetworkToNodeAPI } from "../../utils";
+
```

Comment:
what happens if testnet indexer is down?

Reply:
currently, then tests are failing. Are you suggesting to first make sure indexer is up before running tests? what a good way would be to do it?

Reply:
synced offline - @bowenyang007 will add support for it in the new indexer